src/main.c

```diff
@@ -1,23 +1,23 @@
 #include <stdio.h>
 
+#include "user_interface/user_interface.h"
 #include "game_state/game_state.h"
 
+#include <unistd.h>
+
 int main(int argc, char ** argv) {
   game_state_init();
+  init_ui();
 
-  game_state_on_char_press('a');
-  game_state_on_char_press('l');
-  game_state_on_char_press('e');
-  game_state_on_char_press('x');
-
-  game_state_on_char_press(' ');
-  game_state_on_backspace();
-
-  game_state_on_char_press('i');
+  int i = 0;
 
-  game_state_on_submit();
+  while (1) {
+    ui_get_input();
+    print_ui();
+  }
 
-  print_game_state();
+  // TODO: sigint handler
```

Comment:
The thought here is that we should probably do some memory cleanup on SIGINT. Maybe this isn't strictly necessary but we should at least confirm that there's no nastiness that will make the program crash if SIGINT is sent